lib/debts/get_letters_response.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Debts
+  class GetLettersResponse
+    def initialize(res)
+      # modify response here
```

Comment:
Looks like you are passing in the body as the param in the get_letters method. (May want to rename? -- just a suggestion)

Reply:
i'm passing in the response body so i shortened it to res, what variable name would you suggest?

Reply:
Eh, I think it's fine. Maybe too nitpicky